reckoner/chart.py

```diff
@@ -164,12 +160,13 @@ def update_dependencies(self):
         """ Update the course chart dependencies """
         if self.config.local_development or self.config.dryrun:
             return True
-        logging.debug("Updating chart dependencies: {}".format(self.chart_path))
-        if os.path.exists(self.chart_path):
```

Comment:
I have refactored self.chart_path because it wasn't used anywhere external  and just leveraged repository chart path directly.